src/functions-reference/unbounded_continuous_distributions.Rmd

```diff
@@ -97,11 +97,16 @@ With no logarithm, no subtraction, and no division by a parameter, the
 standard normal log density is much more efficient to compute than the
 normal log density with constant location $0$ and scale $1$.
 
-### Stan Functions
+### Sampling Statement
+
+`y ~ ` **`std_normal`**`(reals y)`
```

Comment:
There's no argument for the sampling statement, so this should just be `y ~ std_normal()`.

Reply:
Thanks!